File: remove_cart.js
const fs = require('fs');
const glob = require('glob'); // Note: run npm install glob if missing, or use basic fs

// Since we may not have glob package, we'll use a basic directory reader
const path = require('path');
const pagesDir = path.join(__dirname, 'pages');

fs.readdir(pagesDir, (err, files) => {
    if (err) throw err;
    files.filter(f => f.endsWith('.html')).forEach(file => {
        const filePath = path.join(pagesDir, file);
        let content = fs.readFileSync(filePath, 'utf8');
        
        let modified = false;
        const lines = content.split('\n');
        const newLines = lines.filter(line => {
            if (line.includes('cart-btn') && (line.includes('fa-shopping-bag') || line.includes('cart.html'))) {
                modified = true;
                return false; // remove this line
            }
            return true;
        });

        if (modified) {
            fs.writeFileSync(filePath, newLines.join('\n'), 'utf8');
        }
    });
    console.log("Removed shopping cart buttons (JS version).");
});

File: remove_cart.py
import os
import glob

html_files = glob.glob('d:/websites/electro-premium/pages/*.html')

count = 0
for path in html_files:
    with open(path, 'r', encoding='utf-8') as f:
        content = f.read()
    
    # We will remove lines that have both cart-btn and a link or an icon, typical of the navbar item.
    lines = content.split('\n')
    new_lines = []
    modified = False
    
    for line in lines:
        if 'cart-btn' in line and ('fa-shopping-bag' in line or 'cart.html' in line):
            modified = True
            continue # Remove this line
        new_lines.append(line)
        
    if modified:
        count += 1
        with open(path, 'w', encoding='utf-8') as f:
            f.write('\n'.join(new_lines))

print(f"Removed shopping buttons from {count} files.")
